packages/gateway/package.json

```diff
@@ -2,19 +2,23 @@
   "name": "@blockful/gateway",
   "version": "1.0.0",
   "description": "",
+  "main": "dist/database.mjs",
+  "types": "dist/database.mjs",
   "scripts": {
     "test": "vitest run",
     "test:watch": "vitest",
-    "start:db": "dotenvx run --env-file=../../.env -- ts-node ./cmd/database.ts",
-    "start:arb": "dotenvx run --env-file=../../.env -- ts-node ./cmd/arbitrum.ts",
-    "dev:db": "ENV_FILE=../../.env nodemon ./cmd/database.ts",
-    "dev:arb": "ENV_FILE=../../.env nodemon ./cmd/arbitrum.ts",
-    "lint": "eslint . --ext .ts --fix"
+    "start:db": "node ./dist/database.mjs",
```

Comment:
Isnt necessary to run the dotenvx command before it anymore?

Reply:
this is for running on the cloud so it expects the environment variables to already be present